index.html

```diff
@@ -34,6 +39,131 @@ <h2>I am a frontend Developer</h2>
       <li class="social-icon"><a href="#"><i class="fab fa-twitter"></i></a></li>
       <li class="social-icon"><a href="#"><i class="fab fa-medium"></i></a></li>
     </ul>
+  </section>
+
+  <!-- Works section -->
+
+  <section class="my-works">
+    <h2 class="works-title">My Recent Works</h2>
+    <hr class="separate">
+    <ul class="projects">
+      <li class="multipost">
+        <img src="img/multipost-Img.jpg" alt="">
```

Comment:
- [ ] [Optional] It would be better if you add a valid alt value, don't just leave it empty, this will increase your site accessibility and SEO